### ISSUE TITLE
Missing enum for creating associations

### ISSUE THREAD

**Issue by deanidean** [HubSpot/hubspot-api-nodejs#475]
In continuation of #466, the auto-generated enum AssociationSpecAssociationCategoryEnum is not exported, but required for creating associations.

**Comment by ryanSN** [comment]
Agree, this make it unusable because we can't even pass in a string https://github.com/HubSpot/hubspot-api-nodejs/blob/bfaa106af7199adbce5f766b1eb35a73f3c72499/codegen/crm/quotes/models/AssociationSpec.ts#L43

**Comment by 4rch1m3d35** [comment]
Im having the same issue.

**Comment by gajus** [comment]
Same issue.

**Comment by gajus** [comment]
CC @ksvirkou-hubspot @atanasiuk-hubspot @asalauyeu-hubspot

Just need to export `AssociationSpecAssociationCategoryEnum` from the package.
